templates/partials/content_row_list_item.html

```diff
@@ -22,7 +26,9 @@
           <i class="fa {{valid ? 'green-dot' : 'red-dot'}} {{verified ? 'fa-check-circle' : 'fa-circle'}}"></i>
         </div>
       </div>
-      <p ng-bind-html="dob | age"></p>
+      <p>
+        <relative-date date="dob" type="age"></relative-date>
+      </p>
       <div class="simprints-stars tier-{{simprintsTier}} {{simprintsTier ? '' : 'ng-hide'}}"></div>
       <div class="detail {{lineage && lineage.length ? '' : 'ng-hide'}}" ng-bind-html="lineage | lineage"></div>
       <div class="warning">{{warning}}</div>
```

Comment:
As mentioned above, we need to stick with our current non-angular approach for now which can only handle expressions and `ng-bind-html` and nothing else.

Reply:
Bummer! I hope migration to React starts soon hehe